package.json

```diff
@@ -12,10 +12,11 @@
     "build": "turbo run build --filter=\"@astrojs/*\"",
     "build:ci": "turbo run build:ci --filter=\"@astrojs/*\"",
     "format:ci": "pnpm run format",
-    "format": "biome check --apply ./ && prettier -w \"**/*\" --ignore-unknown --cache",
+    "format": "biome check --linter-enabled=false --apply ./ && prettier -w \"**/*\" --ignore-unknown --cache",
```

Comment:
Pls help me understand `biome`, isn't there a cli command which only formats, `biome format`?

Reply:
Yeah it does, but it **doesn't** sort the imports. To avail of import sorting, you need the `check` command. With this change, format code and sort the imports. The linter is disabled.